playground/backend/internal/db/local/local_db_test.go

```diff
@@ -0,0 +1,373 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package local_db
+
+import (
+	pb "beam.apache.org/playground/backend/internal/api/v1"
+	"beam.apache.org/playground/backend/internal/db/entity"
+	"beam.apache.org/playground/backend/internal/utils"
+	"context"
+	"testing"
+	"time"
+)
+
+var ctx context.Context
+var localDb *LocalDB
+
+func init() {
+	ctx = context.Background()
+	var err error
+	localDb, err = New()
+	if err != nil {
+		panic(err)
```

Comment:
please use panic only if it is inevitable
https://github.com/uber-go/guide/blob/master/style.md#dont-panic